cmusv-sem/S23SB-SEM-IR
# [SB-109][SB-286] Create ui to allow user to select first responders to transfer

# Description

Allow responders to transfer commander role to other first responders and add new transferred user to group

[commander.mov](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/TR8h4VjidJfKL8LgdRKB/bbdf94e2-d81d-481c-9ed0-b53ade56ac43/commander.mov)****
# Test Plan

Please explain how this change will be tested.



- [x] Integration test (including unit tests with supertest)
- [x] Manual test (Please explain below for the test steps)

1. login as police in browser A
2. login as dispatcher in browser B
3. Open flow for a waiting incident and go to team corner
4. open commender selection and select police 
5. See that only eligible police and me is listed in the selection box.
6. After I select police, confirmation form should show up, and after I click yes, it should show loading and update incident commender to police.
7. on police browser, see that I get notification for the new assigned incident and the new incident will display as my incident 
8. on dispatcher browser incident directory check that incident status is now assigned 


# Task Link

https://cmusem.atlassian.net/browse/SB-286
https://cmusem.atlassian.net/browse/SB-109
